javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,92 @@
+# Short mini project to prepare apprentices for their first JS OOP project
+
+### Projected Time
+
+Total: 65 min- 80min
+- Lesson: 20 min
+- Guided Practice: 15 min
+- Independent Practice: 15 min
+- Materials: 20 min
+
+### Prerequisites
+
+- [Java script lesson 7](/javascript/javascript-7.md)
+
+
+### Objectives
+
+- To  use the new JS object-oriented programming skills to create a few event objects, push them to an array, then use jQuery to display the list of events to an html page.
+
+### Specific Things To Teach
+
+- creating event array.
+- iterating through the array.
+- Displaying array items in HTML page.
+
+### Materials
+
+- [Documentation on each statement of jquery](http://api.jquery.com/jquery.each/)
+- [Arrays in javascript](https://www.w3schools.com/js/js_arrays.asp)
+- [.html function of jquery](https://www.w3schools.com/jquery/html_html.asp)
+
+### Lesson
+
+- Below is the example of code to create a array of events. This needs to be written in a js file. 
+- In order to push more elements to this array you simply use
+".push()" function.
```

Comment:
Make 35 & 36 in one line and it would look good if you write .push like this `.push()` instead of in double quotes...

Reply:
this line has been already excluded.